lib/maintenance_tasks.rb

```diff
@@ -82,4 +82,12 @@ module MaintenanceTasks
   #
   #   @return [String] the name of the parent controller for web UI.
   mattr_accessor :parent_controller, default: "ActionController::Base"
+
+  # @!attribute user_email_method
+  #  @scope class
+  #   The method to call to get the email address of the current user to use when creating a Run.
+  #   This method must be defined on the parent_controller.
+  #
+  #   @return [Symbol] the name of the method to call on the current user.
+  mattr_accessor :user_email_method, default: nil
```

Comment:
If we keep the option, I would definitely bike shed that. Rather than `user`, I'd rather something specific about the role they're playing, so maybe `initiator` or `creator`? And instead of `email`, we should be less specific because nothing forces it to be an email, since it's defined by the application. Implementors may use the field to store a `user id` instead.

Reply:
I'm also leaning towards being less specific but was wondering if we should go even further and just allow some data to be stored in a serialized column. This way applications can decide how to call their actors and even decide to store more context about the actor in the field. A serialized column won't be searchable but I don't think this is something applications will need